tools/hdf_converter/hdf_converter.xml

```diff
@@ -1,7 +1,8 @@
 <tool id="apCLMS_to_RamClustR_converter" name="apLCMS to RamClustR converter" version="0.0.1" python_template_version="3.5">
     <description>converts aplcms HDF output to RamClustR csv input</description>
     <requirements>
-        <container type="docker">maxskorik/hdf_converter</container>
+        <requirement type="package" version="1.1.5">pandas</requirement>
+        <requirement type="package" version="3.6.1">pytables</requirement>
```

Comment:
I'd explicitly declare Python here, you probably get one with pandas/pytables but this way you have no control over version.

Reply:
How do you specifically control that is is a python package?

Reply:
@hechth I don't understand the question, what I am suggesting is to include something like: `<requirement type="package" version="3.7">python</requirement>`